scripts/build-package.ps1

```diff
@@ -24,6 +24,15 @@ function build_nodejs_client {
 # .\nvm-setup.exe /VERYSILENT /SUPRESSMSGBOXES /SP
 # nvm root .\nvm
 
+pwd
+
+cd $Env:USERPROFILE;
+Invoke-WebRequest https://raw.githubusercontent.com/jchip/nvm/v1.5.4/install.ps1 -OutFile install.ps1;
+.\install.ps1 -nvmhome $Env:USERPROFILE\nvm;
+del install.ps1
+
+cd D:\a\aerospike-client-nodejs\aerospike-client-nodejs
```

Comment:
this is run within aerospike-client-nodejs work dir so no need of this hard coded cd.

Reply:
this is run within aerospike-client-nodejs project on already setup windows so we done need these setup